packages/core/src/components/Autocomplete/index.tsx

```diff
@@ -645,15 +650,15 @@ export default class Autocomplete<T extends Item> extends React.Component<Props<
   };
 
   renderNoResults = () => (
-    <MenuRow>
+    <MenuItem>
```

Comment:
`MenuItem` are for things that are clickable. No results shouldn't be.

Reply:
Good catch, thanks, changed these back and added spacing instead